sale_order_type/tests/test_sale_order_type.py

```diff
@@ -88,18 +89,28 @@ def setUp(self):
             }
         )
 
-    def get_sale_order_vals(self):
-        sale_line_dict = {
-            "product_id": self.product.id,
-            "name": self.product.name,
-            "product_uom_qty": 1.0,
-            "price_unit": self.product.lst_price,
-        }
-        return {"partner_id": self.partner.id, "order_line": [(0, 0, sale_line_dict)]}
+    def create_sale_order(self, partner=False):
+        sale_form = Form(self.env["sale.order"])
+        sale_form.partner_id = partner or self.partner
```

Comment:
```suggestion
        sale_form.partner_id = partner or self.partner
        sale_form.date_order = fields.Datetime.now() - timedelta(days=7)
```
Now, show travis error related to https://github.com/OCA/sale-workflow/blob/13.0/sale_automatic_workflow/tests/test_automatic_workflow.py#L54
Maybe defining `date_order` that `sale_automatic_workflow` tests will be checked.